arch/X86/X86MappingInsnOp.inc

```diff
@@ -9811,19 +9811,19 @@
 },
 {	/* X86_SHR64ri, X86_INS_SHR: shr{q}	$dst, $src2 */
 	X86_EFLAGS_MODIFY_OF | X86_EFLAGS_MODIFY_SF | X86_EFLAGS_MODIFY_ZF | X86_EFLAGS_UNDEFINED_AF | X86_EFLAGS_MODIFY_PF | X86_EFLAGS_MODIFY_CF,
-	{ CS_AC_READ | CS_AC_WRITE, CS_AC_IGNORE, 0 }
+	{ CS_AC_READ | CS_AC_WRITE, CS_AC_READ, 0 }
 },
 {	/* X86_SHR8m1, X86_INS_SHR: shr{b}	$dst, 1 */
 	X86_EFLAGS_MODIFY_OF | X86_EFLAGS_MODIFY_SF | X86_EFLAGS_MODIFY_ZF | X86_EFLAGS_UNDEFINED_AF | X86_EFLAGS_MODIFY_PF | X86_EFLAGS_MODIFY_CF,
-	{ CS_AC_READ, CS_AC_IGNORE, 0 }
+	{ CS_AC_READ | CS_AC_WRITE, CS_AC_IGNORE, 0 }
 },
 {	/* X86_SHR8mCL, X86_INS_SHR: shr{b}	$dst, cl */
 	X86_EFLAGS_MODIFY_OF | X86_EFLAGS_MODIFY_SF | X86_EFLAGS_MODIFY_ZF | X86_EFLAGS_UNDEFINED_AF | X86_EFLAGS_MODIFY_PF | X86_EFLAGS_MODIFY_CF,
-	{ CS_AC_READ, CS_AC_READ, 0 }
+	{ CS_AC_READ | CS_AC_WRITE, CS_AC_READ, 0 }
 },
 {	/* X86_SHR8mi, X86_INS_SHR: shr{b}	$dst, $src */
```

Comment:
note that the instructions (such as X86_SHR8mi) has immediate operand, indicated by "i" in its name. and immediate operand should have CS_AC_IGNORE access. therefore, some of your changes are incorrect.

can you please fix them?